src/main/webapp/chat.html

```diff
@@ -2,11 +2,20 @@
 <head>
   <title>Pusher Test</title>
   <script src="https://js.pusher.com/6.0/pusher.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.js"></script>
   <script src="chat.js"></script>
+  <link rel="stylesheet" type="text/css" href="chat.css" />
 </head>
 <body>
-  <h1>"CHAT"</h1>
-  <ul id="chat"></ul>
-  <input type="text" id="chat-input" placeholder="Type message here..."></input>
-  <button onclick="addToChat()">click</button>
+  <div id="chat">
+    <div id="chatbox"></div>
+    <div id="chat-input">
+      <input
+        type="text"
+        id="chat-input-box"
+        placeholder="Type message here..."
+      />
```

Comment:
move this "/>" to line 16.  Indent at least 4 spaces. See section 3.2.2 https://google.github.io/styleguide/htmlcssguide.html